src/assets/text/general.ts

```diff
@@ -38,7 +38,7 @@ export const general = {
   confirm: "Confirm",
   aleoWebsite: "https://aleo.org/",
   pageNotFound: "404:Page not found",
-  countdown: (val: string): string => `${val} left`,
+  countdown: (val: string): string => `${val}`,
```

Comment:
What's the point in having this function if it does nothing?

Reply:
hmmm yeah since I removed the left, this is actually not necessary anymore. Coolio